change/@microsoft-teams-js-ab0f025f-cb65-450c-92a8-8000cb58057b.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Added new sub capability `pages.self`.",
```

Comment:
I might briefly mention what this sub-capability enabled (navigating within an app without specifying the appid)